lib/internal/child_process.js

```diff
@@ -692,7 +692,7 @@ function setupChannel(target, channel, serializationMode) {
     const obj = handleConversion[message.type];
 
     // Update simultaneous accepts on Windows
-    if (process.platform === 'win32') {
+    if (handle.setSimultaneousAccepts && process.platform === 'win32') {
```

Comment:
Could you make the check `typeof handle.setSimultaneousAccepts === 'function'` here and below.

Reply:
Done !